recipes/ctlearn/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "ctlearn" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ctlearn-{{ version }}.tar.gz
+  sha256: 1e0429e0ae20e35adeed9445d482944064798a8e1962bc97d77c520256672c39
+  
+build:
+  entry_points:
+    - ctlearn = ctlearn.run_model:main
+    - build_irf = ctlearn.build_irf:main
+  #noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win or not py==310]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
But If I add noarch I cannot add the skip part, do I add it and assume that Windows will fail and we can merge the PR? It fails because of tensorflow version, that is not supported in windows since version 1.14 (now it is in 2.16*)

Reply:
You don't need to skip. As I mentioned above let Windows fail, we will merge, if/when tensorflow is available on Windows, this package will be ready without the need for any modifications.

Reply:
Perfect, I have made the necessary changes. Thanks